resolwe_bio/processes/import_data/annotation.yml

```diff
@@ -171,7 +204,21 @@
     program: |
       re-import {{ src.file_temp }} {{ src.file }} "gtf|gff" "gtf" 0.3
 
+      igvtools sort "${NAME}".gtf "${NAME}"_sorted.gtf
+      re-checkrc "Sorting of GTF file failed."
+      igvtools index "${NAME}"_sorted.gtf || re-warning "Indexing of GTF file failed."
```

Comment:
Is this a common case (e.g. for igvtools index to fail)? Could you write a short comment why it fails, etc...?

Reply:
These are input data that we can't have a validation on it. Some GTF files, that are not in standard GTF format are ok to work with specific tools, but are not acceptable by IGV. In these cases the user will be able to upload and use GTF files, but not visualise annotation in IGV. This will most probably happen vary rare cases.